modules/web-console/frontend/app/types/Agent.ts

```diff
@@ -69,3 +72,15 @@ export type VisorQueryResult =
 export type QuerySqlX2Response = VisorQueryResult
 
 export type QueryFetchFirstPageResult = VisorQueryResult
+
+export enum IgniteFeatures {
+	INDEXING= 15,
+	WC_ROLLING_UPGRADE_STATUS = 20,
+	WC_SNAPSHOT_CHAIN_MODE = 21
```

Comment:
Where WC_DR_STATE_EVENTS ?

Reply:
Will do this in separated ticket

Reply:
@nva please make a task for this so we don't forget.